lib/mini_profiler/config.rb

```diff
@@ -39,7 +39,8 @@ def self.default
           @skip_sql_param_names = /password/ # skips parameters with the name password by default
           @enable_advanced_debugging_tools = false
           @snapshot_every_n_requests = -1
-          @snapshots_limit = 1000
```

Comment:
There's a section in the readme that uses this configuration: https://github.com/MiniProfiler/rack-mini-profiler#snapshots-sampling

Will you be updating its usage in this PR?

Reply:
Yes, I will! Thanks for pointing it out!

Reply:
Updated!